tests/io-app.yaml

```diff
@@ -0,0 +1,49 @@
+# -*- mode: yaml -*-
+---
+kind: PersistentVolumeClaim
+apiVersion: v1
+metadata:
+  name: pv-io
+spec:
+  storageClassName: kadalu.replica3
+  accessModes:
+    - ReadWriteMany
+  resources:
+    requests:
+      storage: 10Mi
+
+---
+apiVersion: apps/v1
+kind: Deployment
+metadata:
+  name: io-app
+spec:
+  replicas: 2
+  selector:
+    matchLabels:
+      app: io-app
+  template:
+    metadata:
+      labels:
+        app: io-app
+    spec:
+      containers:
+      - name: io-pod
+        image: docker.io/leelavg/dark-side:latest
```

Comment:
you want to actually call it `dark-side`? or any particular name?

Reply:
Not at all 😅, please choose any name for the docker image before pushing to kadalu docker hub. 

You can also edit any labels in Dockerfile as well, just let me know of the edits I'll edit yaml and Dockerfile accordingly.

Reply:
There are 2 thoughts I have!

1. Always build locally and use the same - May add just 3-4 mins extra in complete test time, which is OK. Benefit is, any changes in `tests/Dockerfile` also gets tested. (or add this build locally as a conditional step, only if the particular file changed in PR).

2. The PR merge job can look for any changes to `tests/Dockerfile` and then build an image (for now only `linux/amd64`) and push it to docker hub. As we already have secret and everything properly setup, this works best instead of manually pushing the image.

Reply:
Ack.

1. I can go with adding a conditional step for local build
2. This image is similar to sample-app in the sense gets updated very rare, will push conditionally based on file in PR